### PR TITLE
Provide an ability to track multiple hosts

### PR DESCRIPTION
**Pull request by vharitonsky**
--

### CODE REVIEW

File: cmd/mtr-exporter/main.go [hunk 1]
```diff
@@ -6,13 +6,14 @@ package main
 import (
 	"flag"
 	"fmt"
+	"github.com/robfig/cron/v3"
```
Comment (mgumz) [inline cmd/mtr-exporter/main.go:9]: please keep the external dependencies on its extra section like it was.

File: cmd/mtr-exporter/main.go [hunk 2]
```diff
@@ -6,13 +6,14 @@ package main
...
 	"log"
 	"net/http"
 	"os"
-
-	"github.com/robfig/cron/v3"
 )
 
+var hosts arrayFlags
```
Comment (mgumz) [inline cmd/mtr-exporter/main.go:15]: you introduce a "global" variable while the existing code tries to minimize that. in general it is favourable to adjust to the existing style.

**Comment by mgumz** [comment]
try https://github.com/mgumz/mtr-exporter/tree/feature/multi-jobs if you want. it implements the idea i outlined above.

**Comment by vharitonsky** [comment]
Thanks for you extended commend, I just wanted a quick and simple way to add multiple host to deploy the project on k8s and track all our internal inftrastructure from a single pod. Multi job feature seems well suited for this.

**Comment by mgumz** [comment]
@vharitonsky if you are in the mood: https://github.com/mgumz/mtr-exporter/commits/feature/multi-jobs - i have brought it up to `master` level and will release a 0.3.0 most likely very soon :)

feedback on if its working well in your k8s environment is welcome

**Comment by mgumz** [comment]
thanks for the effort, much appreciated.

still, I am closing this PR here and go with branch https://github.com/mgumz/mtr-exporter/tree/feature/multi-jobs.

**Pull request closed without merging**